pkg/api/conn_track.go

```diff
@@ -29,7 +27,7 @@ type ConnTrack struct {
 	KeyDefinition        KeyDefinition `yaml:"keyDefinition,omitempty" doc:"fields that are used to identify the connection"`
 	OutputRecordTypes    []string      `yaml:"outputRecordTypes,omitempty" enum:"ConnTrackOutputRecordTypeEnum" doc:"output record types to emit"`
 	OutputFields         []OutputField `yaml:"outputFields,omitempty" doc:"list of output fields"`
-	EndConnectionTimeout time.Duration `yaml:"endConnectionTimeout,omitempty" doc:"duration of time to wait from the last flow log to end a connection"`
+	EndConnectionTimeout Duration      `yaml:"endConnectionTimeout,omitempty" doc:"duration of time to wait from the last flow log to end a connection"`
```

Comment:
duration in time in seconds?

Reply:
Not necessary. The config value is passed to `time.ParseDuration()` during unmarshalling.
https://pkg.go.dev/time#ParseDuration
So for example `30s` will be parsed as 30 seconds and `2m` will be parsed as 2 minutes.